Frameworks/AEPSwiftUI/Sources/Constants.swift

```diff
@@ -15,7 +15,12 @@ import SwiftUI
 
 enum Constants {
     static let LOG_TAG = "AEPSwiftUI"
-    static let EXTENSION_VERSION = "5.1.0"
+    static let EXTENSION_VERSION = "5.1.1"
+
+    enum Cache {
+        static let CACHE_NAME = "com.adobe.messaging.contentcard.ui.cache"
```

Comment:
Thoughts on this? 
Should I change the location to be different than messaging ?